schema/configuration.go

```diff
@@ -57,6 +57,67 @@ type DataSharing struct {
 
 var MalformedConfigError error = errors.New("config file is malformed")
 
+func DefaultWalletsConfig() *WalletsConfig {
+	var feeAPI = "https://btc.fees.openbazaar.org"
+	return &WalletsConfig{
+		BTC: &CoinConfig{
+			Type:             WalletTypeAPI,
+			APIPool:          CoinPoolBTC,
+			APITestnetPool:   CoinPoolTBTC,
+			FeeAPI:           feeAPI,
+			LowFeeDefault:    1,
+			MediumFeeDefault: 10,
+			HighFeeDefault:   50,
+			MaxFee:           200,
+			WalletOptions:    nil,
+		},
+		BCH: &CoinConfig{
+			Type:             WalletTypeAPI,
+			APIPool:          CoinPoolBCH,
+			APITestnetPool:   CoinPoolTBCH,
+			FeeAPI:           feeAPI,
```

Comment:
I think `feeAPI` for everything other than BTC should bet set to "". If it isn't the wallet may try to use that API.